contracts/vaults/YearnVaults/base/RibbonVault.sol

```diff
@@ -600,6 +601,28 @@ contract RibbonVault is OptionsVaultYearnStorage, DSMath {
         }
     }
 
+    /*
+      Upgrades the vault to point to the latest yearn vault for the asset token
+    */
+    function upgradeYearnVault() external onlyOwner {
```

Comment:
this should be called in between commitAndClose() and rollToNextOption(). only time we have all the collateral back in the vault